vaadin-testbench-unit/src/main/java/com/vaadin/flow/component/combobox/ComboBoxWrap.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright (C) 2022 Vaadin Ltd
+ *
+ * This program is available under Commercial Vaadin Developer License
+ * 4.0 (CVDLv4).
+ *
+ *
+ * For the full License, see <https://vaadin.com/license/cvdl-4.0>.
+ */
+package com.vaadin.flow.component.combobox;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.vaadin.flow.component.ItemLabelGenerator;
+import com.vaadin.flow.data.provider.DataCommunicator;
+import com.vaadin.flow.data.provider.Query;
+import com.vaadin.flow.function.SerializableConsumer;
+import com.vaadin.testbench.unit.ComponentWrap;
+import com.vaadin.testbench.unit.Wraps;
+import com.vaadin.testbench.unit.internal.BasicUtilsKt;
+
+@Wraps(fqn = "com.vaadin.flow.component.combobox.ComboBox")
+public class ComboBoxWrap<T extends ComboBox<Y>, Y> extends ComponentWrap<T> {
+    /**
+     * Wrap given component for testing.
+     *
+     * @param component
+     *         target component
+     */
+    public ComboBoxWrap(T component) {
+        super(component);
+    }
+
+    /**
+     * Simulate writing a filter to the combobox.
+     * <p/>
+     * Use {@link #getSuggestions()} to get the string values show in the
+     * dropdown or
+     * {@link #getSuggestionItems()} to get the actual items in the suggestion.
+     *
+     * @param filter
+     *         string to use for filtering
+     */
+    public void setFilter(String filter) {
+        ensureComponentIsUsable();
+        try {
+            final Field filterSlot = ComboBox.class.getDeclaredField(
+                    "filterSlot");
+            final boolean state = filterSlot.canAccess(getComponent());
+            filterSlot.setAccessible(true);
+            ((SerializableConsumer<String>) filterSlot.get(
+                    getComponent())).accept(filter);
+            filterSlot.setAccessible(state);
+        } catch (NoSuchFieldException | IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    /**
+     * Select item by client string representation.
+     *
+     * @param selection
+     *         item representation string
+     */
+    public void selectItem(String selection) {
```

Comment:
Is `null` a valid input for this method? Perhaps to clear current selection (if it is possible)?